src/serac/physics/elasticity.cpp

```diff
@@ -124,6 +124,8 @@ void Elasticity::QuasiStaticSolve()
   K_inv_.Mult(*bc_rhs_, displacement_.trueVec());
 }
 
+mfem::Operator& Elasticity::getOutputGradient() { return *K_mat_; }
```

Comment:
Any specific reason for the "Output" naming? Same with the Residual.

Reply:
I was trying to match the  `getOutput()` notion. I'm open to better naming.

Reply:
I changed the name just to `outputGradient` and `outputResidual` just to better match `outputState()`.